editor/src/components/navigator/navigator-item/navigator-item-dnd-container.tsx

```diff
@@ -236,16 +236,15 @@ export class NavigatorItemDndWrapper extends PureComponent<
     }
   }
 
-  render() {
+  render(): React.ReactElement {
     const props = this.props
 
     const navigatorItemContainer = (
       <div
         key='navigatorItem'
+        id={`navigator-item-${TP.toVarSafeComponentId(this.props.templatePath)}`}
```

Comment:
does react-window rely on the `id`?

Reply:
Nope, this was just helpful in debugging.